### PR TITLE
Auth UI for Angular template

### PR DESCRIPTION
**Pull request by ovchinnikov**

### CODE REVIEW

File: packages/devextreme-cli/testing/app-template.test.shared.js [hunk 1]
```diff
@@ -168,7 +168,7 @@ module.exports = (env) => {
                                         return;
                                     }
 
-                                    const name = env.engine === 'react' ? 'login-react' : 'login';
+                                    const name = env.engine === 'vue' ? 'login' : 'login-react';
```
Comment (nightskylark) [inline packages/devextreme-cli/testing/app-template.test.shared.js:171]: It looks strange to use 'react' word in angular approach. Maybe rename 'login' and 'login-react' to 'login-old' and 'login'? So we'll get rid of the 'login-old' after Auth UI will be implemented for Vue.

**Comment by nightskylark** [comment]
Did you test a Template generater? Do we need to update config files after these changes?

**Comment by ovchinnikov** [comment]
> Did you test a Template generater? Do we need to update config files after these changes?

What is template generator? Which config files are you talking about?

**Comment by nightskylark** [comment]
> > Did you test a Template generater? Do we need to update config files after these changes?
> 
> What is template generator? Which config files are you talking about?

https://github.com/DevExpress/devextreme-cli/tree/master/templates-generator

**Comment by ovchinnikov** [comment]
> > > Did you test a Template generater? Do we need to update config files after these changes?
> > 
> > 
> > What is template generator? Which config files are you talking about?
> 
> https://github.com/DevExpress/devextreme-cli/tree/master/templates-generator

I don't know what are these files for. Do we have documentation for it? How should I test it. How should I understand do we need to change these file or not?

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**